bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/loader/AbstractScriptFileWatcher.java

```diff
@@ -353,18 +358,14 @@ private boolean createAndLoad(ScriptFileReference ref) {
         return false;
     }
 
-    private void initialImport() {
-        File directory = watchPath.toFile();
```

Comment:
Why did you remove the directory creation?

Reply:
I thought it's redundant because:
- The directory creation is also done in WatchServiceImpl
- The directory creation is also done in `activate()` - this is also redundant here, but happy to leave it there to ensure this behaviour in case it changed in WatchServiceImpl.
- Prior to this PR, the directory creation is checked/performed inside `initialImport`

There is only a tiny window between activate() and the time the final call to `initialImport()` was made upon reaching startlevel 100 that an existing directory ensured by `activate()` got deleted. However, I'm happy to restore it if you think it's essential.

Reply:
I've tested deleting conf/automation/jsr223 then starting openhab up. It got recreated, and it monitors changes in the directory. I tried creating a script in there and it got (re)loaded. So I believe it's fine to remove it from initialImport.